docs/graphql/manual/remote-schemas/index.rst

```diff
@@ -127,3 +127,59 @@ Query your remote server by making requests to the Hasura graphql endpoint (``/v
 
 .. _Interfaces: https://graphql.github.io/learn/schema/#interfaces
 .. _Unions: https://graphql.github.io/learn/schema/#union-types
+
+
+Bypassing Hasura's authorization system for remote schema queries
+-----------------------------------------------------------------
+
+It might be necessary sometimes to bypass Hasura's authorization system (calling
+the configured webhook, or validating the JWT), for queries that are for a
+remote GraphQL server.
+
+**For example**, you have a remote GraphQL server which does authentication,
+i.e. signup and login, and you have added it as a remote schema. In this case,
+you would not want to perform Hasura's authorization when the user is making a
+login/signup request.
+
+There is no first-class option to currently do this via any configuration in
+Hasura. However a similar solution can achieved by the following workarounds:
+
+Bypassing webhook authorization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you have a :doc:`webhook authorization setup <../auth/webhook>`, in the
+normal scenario, your authorization webhook should return ``200`` on success and
+``401`` if it is unable to authorize the current request or authorization
+information is absent (like cookie, authorization header etc.)
+
+In the workaround, the webhook should respond with ``200`` and ``x-hasura-role:
+anonymous`` instead of a ``401`` when the authorization information is absent or
+it fails to resolve it. And, when adding the remote schema, the ``Forward all
+headers from client`` option is checked. So the remote server will get the
```

Comment:
@ecthiender This includes all the headers passed to the webhook, right? Including say, an `Authorization` header.